### PR TITLE
Added camera modes and organized variables

### LINKED ISSUE #93: Limelight should switch between camera and vision
Can the Limelight programmatically switch between modes?

### PR DESCRIPTION
**Pull request by NWSchallock**
PTAL

Fixes #93

### CODE REVIEW

File: src/main/cpp/subsystems/Chassis.cpp [hunk 1]
```diff
@@ -98,3 +100,11 @@ double Chassis::GetLeftVoltage() {
 double Chassis::GetRightVoltage() {
   return right2Wheel.GetAppliedOutput();
 }
+
+void Chassis::SetCamVisionProcessing() {
+  mLimeLightTable->PutNumber("camMode", 0);
+}
+
+void Chassis::SetCamDriverCam() {
```
Comment (duane701) [inline src/main/cpp/subsystems/Chassis.cpp]: How do we test this from the smartdashboard?

File: src/main/cpp/commands/SetControlDrive.cpp [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#include "commands/SetControlDrive.h"
+#include "commands/TankDrive.h"
+#include "commands/SetCamModeDriver.h"
+
+SetControlDrive::SetControlDrive() {
+  AddParallel(new SetCamModeDriver());
```
Comment (duane701) [inline src/main/cpp/commands/SetControlDrive.cpp]: These commands are in the same subsystem. They wont be parallel.

Change to sequential.

File: src/main/cpp/commands/SetVisionDrive.cpp [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#include "commands/SetVisionDrive.h"
+#include "commands/SetCamModeVision.h"
+#include "commands/VisionDrive.h"
+
+SetVisionDrive::SetVisionDrive() {
+  AddParallel(new SetCamModeVision());
```
Comment (duane701) [inline src/main/cpp/commands/SetVisionDrive.cpp]: Change to sequential.

File: src/main/cpp/subsystems/Chassis.cpp [hunk 4]
```diff
@@ -65,8 +65,10 @@ void Chassis::SetArcadeDrive(double speed, double rotation) {
 }
 
 double Chassis::GetVisionRotation() {
-  constexpr double pRotation = 1 / 27;
-  constexpr double setPoint = 0;
+  constexpr double maxSpeed {0.5};
+  constexpr double maxAngle {27.0};
+  constexpr double pRotation {maxSpeed / maxAngle};
+  constexpr double setPoint {0};
```
Comment (duane701) [inline src/main/cpp/subsystems/Chassis.cpp]: FYI -- these changes should have been in a different commit.

**Comment by NWSchallock** [comment]
PTAL

**Pull request merged**